File: Car-Dealership-App/node/models/car.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;

const carSchema = new Schema({
  owner: {
    type: String,
    trim: true,
    required: "Enter owner of car",
  },
  make: {
    type: String,
    trim: true,
    required: "Enter make of car",
  },
  model: {
    type: String,
    trim: true,
    required: "Enter model of car",
  },
  year: {
    type: Number,
    trim: true,
    required: "Enter year of car",
  },
  color: {
    type: String,
    trim: true,
    required: "Enter color of car",
  },
  price: {
    type: Number,
    required: "Enter asking price",
  },
  image: {
    type: String,
    trim: true,
  },
  date: {
    type: Date,
    default: Date.now,
  },
});

const Listing = mongoose.model("Listing", carSchema);

module.exports = Listing;

File: Car-Dealership-App/dotnetwebapi/Models/Car.cs
using System;

namespace dotnetwebapi.Models
{
    public class Car
    {
        public int CarId { get; set; }
        public string Owner { get; set; }
        public string Make { get; set; }
        public string Model { get; set; }
        public int Year { get; set; }
        public string Color { get; set; }
        public double Price { get; set; }
        public string Image { get; set; }
        public DateTime Date { get; set; }
    }
}